src/main/java/mate/academy/spring/validator/Constraint.java

```diff
@@ -0,0 +1,20 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package mate.academy.spring.validator;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.ConstraintValidator;
+
+@Documented
+@Target({ElementType.ANNOTATION_TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Constraint {
```

Comment:
where do you use this annotation? do we need to create this class if you want to override default @Email annotation checking logic?

let's better create our own @Email annotation for this task